CONTRIBUTING.md

```diff
@@ -46,13 +46,36 @@ See the [contributor's testing guide](https://github.com/kudobuilder/kudo/blob/m
 
 ## Community, Discussion, and Support
 
-Learn how to engage with the Kubernetes community on the [community page](https://kudo.dev/community/).
+Learn how to engage with the KUDO community on the [community page](https://kudo.dev/community/).
 
 You can reach the maintainers of this project at:
 
 - [Slack](https://kubernetes.slack.com/messages/kudo/)
 - [Mailing List](https://groups.google.com/d/forum/kudobuilder)
 
+## Code culture
+
+This is a set of practices we try to live by when developing KUDO. These are just defaults (soft rules). Deviations from them are possible, only has to be justified.
+
+### General guidelines
+- Master is always releasable (green CI)
+- All feature/bug-fixing work should have an open issue with a description
+- Every user-facing feature that is NOT behind a feature gate should have an integration test
+
+### Pull requests
+- One core-team member has to approve the PR to be able to merge (defined in code owners file)
+- When core-team member puts 'request for changes' on your PR, that stops merging until it's resolved (even when have enough approvals)
+- Since KUDO is developed in multiple timezones, try to keep the PR open for everyone to be able to see it (~24h)
```

Comment:
Should we mention public holidays?

Reply:
Do we need to be explicit? I like guidelines/light rules :) we're fairly senior team right now, I would be ok with letting everyone interpret what does it mean for the other timezone to see their code :) but I can surely add it if you feel like it helps

Reply:
Agreed, I'm :+1: on not needing to be explicit here...I think the intent of the culture is giving opportunities to others, and as long as we carry that intent, we can all be roughly on the same page. Just something to keep in mind before you hit that squash and merge button - have there been opportunities for my fellow KUDO developers to review this? If I'm merging without, am I justifying why this needs to be in without giving that opportunity (say, a bug fix before people in US times have had a chance to see it), etc. Nothing hard and fast.

Reply:
I don't mean enumerating holidays, I'm also all for loose guidelines.

I thought about this as a helpful hint, because personally I often forget that people on the other side of the globe might be away for a while.

The mention of just `24h` here would further cement my conviction that passage of time is equivalent to providing such opportunities for review.